index.html

```diff
@@ -26,7 +26,7 @@ <h1>Simple Calculator App</h1>
                     It is a HTML/CSS/JavaScript based application capable of running in your browser. It can be used to
                     perform simple arithmetic operations involving two operands and a single operator.
                     <br /><br />
-                    You can check out the hosted version <a href="/src/" target="_blank">here</a>.
+                    You can check out the hosted version <a href="https://iampranabroy.github.io/osse-calculator/src/" target="_blank">here</a>.
```

Comment:
Hardcoded the proper link here so that the hosted source code opens from the GitHub page directly